apps/ligo-virgo/src/pages/workspace/docs/Page.tsx

```diff
@@ -22,16 +22,29 @@ import { type BlockEditor } from '@toeverything/components/editor-core';
 import { useFlag } from '@toeverything/datasource/feature-flags';
 import { CollapsiblePageTree } from './collapsible-page-tree';
 import { Tabs } from './components/tabs';
+import { TabMap, TAB_TITLE } from './components/tabs/Tabs';
+import { TOC } from './components/toc';
 import { WorkspaceName } from './workspace-name';
+
 type PageProps = {
     workspace: string;
 };
 
 export function Page(props: PageProps) {
+    const [activeTab, setActiveTab] = useState(
+        TabMap.get(TAB_TITLE.PAGES).value
+    );
     const { page_id } = useParams();
     const { showSpaceSidebar, fixedDisplay, setSpaceSidebarVisible } =
         useShowSpaceSidebar();
     const dailyNotesFlag = useFlag('BooleanDailyNotes', false);
+    const editorRef = useRef(null);
+
+    const onTabChange = v => setActiveTab(v);
+
+    const getEditor = editor => {
+        editorRef.current = editor;
+    };
```

Comment:
It's not a good pattern

Reply:
Because EditorContainer and TOC is separate, there maybe tow idea to fix it:
1. Provider
2. Callback

I tend to use callback, less code and clearer.

Reply:
Current, useCurrentEditors hook contains some special logic, use it can't get editor-case in TOC.

I will use callback make TOC work this moment. 
After get this logic, replace callback.